hack/sync-manifests.sh

```diff
@@ -0,0 +1,115 @@
+}
+
+update_crds() {
+    # Copy all CRD files to destination directory.
+    # shellcheck disable=SC2086
+    find ${SRC_CRD_DIR} -type f -name "*.yaml" ${FIND_EXCLUDE_ARGS} -exec cp {} ${DST_CRD_DIR} \;
+}
+
+update_rbac() {
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_RBAC_DIR}/serviceaccount.yaml
+    yq -e "select(.kind == \"ServiceAccount\" and .metadata.name == \"kubeflow-trainer-controller\") | ${YQ_ARGS}" "${MANIFESTS_FILE}" >> ${DST_RBAC_DIR}/serviceaccount.yaml
+
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_RBAC_DIR}/clusterrole.yaml
+    yq -e "select(.kind == \"ClusterRole\" and .metadata.name == \"kubeflow-trainer-controller\") | ${YQ_ARGS}" "${MANIFESTS_FILE}" >> ${DST_RBAC_DIR}/clusterrole.yaml
+
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_RBAC_DIR}/clusterrolebinding.yaml
+    yq -e "select(.kind == \"ClusterRoleBinding\" and .metadata.name == \"kubeflow-trainer-controller\") | ${YQ_ARGS}" "${MANIFESTS_FILE}" >> ${DST_RBAC_DIR}/clusterrolebinding.yaml
+
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_RBAC_DIR}/role.yaml
+    yq -e "select(.kind == \"Role\" and .metadata.name == \"kubeflow-trainer-controller\") | ${YQ_ARGS}" "${MANIFESTS_FILE}" >> ${DST_RBAC_DIR}/role.yaml
+
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_RBAC_DIR}/rolebinding.yaml
+    yq -e "select(.kind == \"RoleBinding\" and .metadata.name == \"kubeflow-trainer-controller\") | ${YQ_ARGS}" "${MANIFESTS_FILE}" >> ${DST_RBAC_DIR}/rolebinding.yaml
+}
+
+update_controller() {
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_CONTROLLER_DIR}/deployment.yaml
+    yq -e "select(.kind == \"Deployment\" and .metadata.name == \"kubeflow-trainer-controller\") | ${YQ_ARGS}" "${MANIFESTS_FILE}" >> ${DST_CONTROLLER_DIR}/deployment.yaml
+
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_CONTROLLER_DIR}/service.yaml
+    yq -e "select(.kind == \"Service\" and .metadata.name == \"kubeflow-trainer-controller-service\") | ${YQ_ARGS}" "${MANIFESTS_FILE}" >> ${DST_CONTROLLER_DIR}/service.yaml
+}
+
+update_webhook() {
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_WEBHOOK_DIR}/secret.yaml
+    yq -e "select(.kind == \"Secret\" and .metadata.name == \"kubeflow-trainer-webhook-cert\") | ${YQ_ARGS}" "${MANIFESTS_FILE}" >> ${DST_WEBHOOK_DIR}/secret.yaml
+
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_WEBHOOK_DIR}/validatingwebhookconfiguration.yaml
+    yq -e "select(.kind == \"ValidatingWebhookConfiguration\" and .metadata.name == \"validator.trainer.kubeflow.org\" | ${YQ_ARGS})" "${MANIFESTS_FILE}" >> ${DST_WEBHOOK_DIR}/validatingwebhookconfiguration.yaml
+}
+
+update_runtimes() {
+    printf '#\n%s\n#\n\n' "${LICENSE}" > ${DST_RUNTIMES_DIR}/pretraining/torch-distributed.yaml
+    yq -e "select(.kind == \"ClusterTrainingRuntime\" and .metadata.name == \"torch-distributed\")" "${MANIFESTS_FILE}" > ${DST_RUNTIMES_DIR}/pretraining/torch-distributed.yaml
+}
+
+cleanup() {
+    rm -f "${MANIFESTS_FILE}"
+}
+
+setup
+update_crds
+update_rbac
+update_controller
```

Comment:
Please can we keep the controller_manager name for it:
```suggestion
update_controller_manager
```